modules/exploits/linux/http/artica_proxy_rce.rb

```diff
@@ -0,0 +1,136 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Artica proxy 4.30.000000 Remote Code Execution',
```

Comment:
The module name should describe the vulnerability and roughly match the file name.

`remote code/command execution` is implied as this is an `exploit` module. In this instance, the specific type of RCE is command injection.

Given that the command injection occurs in `service-cmds-peform` and the module also uses an auth bypass; suggest something like this:

```suggestion
        'Name' => 'Artica Proxy 4.30.000000 Auth Bypass service-cmds-peform Command Injection',
```

With file name:

* artica_proxy_auth_bypass_service_cmds_peform_command_injection